### PR TITLE
add eslint-config-scnu

### PR DESCRIPTION
**Pull request by kok-s0s**

### CODE REVIEW

**Review by YXS-2000 — APPROVED** [review]

**Comment by kok-s0s** [comment]
eslint-config-scnu --> https://github.com/kok-s0s/eslint-config-scnu

**Review by okkjoo — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,8 @@
 {
-    "editor.detectIndentation": false
-}
\ No newline at end of file
+	"editor.detectIndentation": false,
+
+	"prettier.enable": false,
```
Comment (clean99) [inline .vscode/settings.json:4]: could we turn on prettier? we may need prettier to handle indentation, etc.

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,8 @@
 {
-    "editor.detectIndentation": false
-}
\ No newline at end of file
+	"editor.detectIndentation": false,
```
Comment (clean99) [inline .vscode/settings.json:2]: could we make indentation 2 space

File: package.json [hunk 3]
```diff
@@ -1,53 +1,54 @@
...
+			"not op_mini all"
+		],
+		"development": [
+			"last 1 chrome version",
+			"last 1 firefox version",
+			"last 1 safari version"
+		]
+	},
+	"devDependencies": {
+		"eslint-config-scnu": "^1.3.1",
```
Comment (clean99) [inline package.json]: could add documentation about our lint tool(can do it later):
1. how to config and use it?
2. what is the style/format this tool take?

**Review by clean99 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (kok-s0s) [reply package.json]: ok 🌟

**Pull request closed without merging**